src/components/CheckBox/CheckBox.tsx

```diff
@@ -71,11 +99,12 @@ export const CheckBox: FC<CheckboxProps> = React.forwardRef(
             >
                 <input
                     ref={ref}
+                    aria-disabled={allowDisabledFocus}
```

Comment:
I believe we are doing this wrong. Based on docs, this should mimic the disabled value
 https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled
 
 In which case this might look like such:
 aria-disabled={disabled}
 disabled={!allowDisabledFocus && disabled}

and then styling for disabled state would be based on the value of the aria-disabled attribute... thoughts?

Reply:
Will update.